waves-integration/src/main/scala/com/wavesplatform/dex/grpc/integration/clients/combined/CombinedWavesBlockchainClient.scala

```diff
@@ -259,7 +259,8 @@ object CombinedWavesBlockchainClient extends ScorexLogging {
     wavesBlockchainClientSettings: WavesBlockchainClientSettings,
     matcherPublicKey: PublicKey,
     monixScheduler: Scheduler,
-    grpcExecutionContext: ExecutionContext
+    grpcExecutionContext: ExecutionContext,
+    mkCombinedStream: => (MatcherExtensionClient, BlockchainUpdatesClient) => CombinedStream
```

Comment:
why we need by name modifier for args here?

Reply:
Hm, the old code ) I'll change this.

Reply:
Fixed